src/ert/gui/simulation/view/progress.py

```diff
@@ -60,32 +60,26 @@ def paint(self, painter, option: QStyleOptionViewItem, index: QModelIndex) -> No
         if data is None:
             return
 
-        nr_reals = data["nr_reals"]
-        status = data["status"]
+        nr_reals: int = data["nr_reals"]
+        status: dict[str, int] = data["status"]
```

Comment:
This typehint doesn't work on python 3.8 without `from __future__ import annotations`.